app/activity/[title]/page.tsx

```diff
@@ -0,0 +1,96 @@
+'use client'
+import {
+  Heading,
+  Box,
+  Container,
+  useColorModeValue,
+  Flex,
+  Spacer,
+  Text,
+  Textarea,
+  Input,
+  FormControl,
+  FormLabel
+} from '@chakra-ui/react'
+import { FiChevronLeft, FiClock, FiMapPin, FiLink2 } from 'react-icons/fi'
+import { Carousel } from '@/components/carousel'
+import { Link } from '@/components/link'
+import { Header, Footer } from '@/components/navigation'
+import { TrashIcon, EditIcon } from '@/icons'
+import { customColors } from '@/theme/color'
+
+export default function ActivityDetails({ params }: { params: { title: string } }) {
+  const bg = useColorModeValue('white', 'gray.800')
+  const color = useColorModeValue('black', 'gray.300')
+  return (
+    <>
+      <Header />
+      <Box as="main" minH="100vh" bg={bg} color={color}>
+        <Container
+          maxW={{ base: '710px' }}
+          pt={{ base: '20px', md: '30px' }}
+          pb={{ base: '40px', md: '80px' }}
+        >
+          <Flex pb={{ base: '30px', md: '40px' }}>
+            <Heading fontSize={{ base: 'xl', md: '2xl' }}>
+             {params.title} Asakusa Temple
+            </Heading>
+            <Spacer />
+            <TrashIcon />
+            <EditIcon />
+          </Flex>
+          <Flex flexDirection="column" pb={{ base: '40px', md: '48px' }}>
+            <Box display="flex" alignItems="center" my={1}>
+              <FiClock style={{ color: customColors.gray[400] }} />
+              <Text
+                fontSize={{ base: 'md', md: 'lg' }}
+                fontWeight="semibold"
+                mx={2}
+              >
+                Oct.1, 2023 16:00
+              </Text>
+            </Box>
+            <Box display="flex" alignItems="center" my={1}>
+              <FiMapPin style={{ color: customColors.gray[400] }} />
+              <Text
+                fontSize={{ base: 'md', md: 'lg' }}
+                fontWeight="semibold"
```

Comment:
I used the number.. Which we should use? 👀

Reply:
I prefer using those variables since we've employed the same variables in Figma. It makes it easy for me to identify which font size to use. What are your thoughts on that?

Reply:
Ok! I think it's good, Then I will fix at my part later :)